2_large_language_models/README.md

```diff
@@ -1 +1,30 @@
 # Weeks 4 & 5 - Large Language Models
+
+The first ML project we'll be developing in the LLM section is a word similarity machine.
+
+The second project will be to create a system for predicting sentiment analysis, given a sentence.
+
+## Project 1 - Word Similarity
+
+For this week, we'll go through the process of creating a word similarity machine. This is a system that can take two words and determine how similar they are. For example, the words "cat" and "dog" are very similar, while the words "cat" and "banana" are not.
```

Comment:
Do you want to use the term "application" or "service" instead of "machine"?

Reply:
Switched to application